doc/manual/datasets_index.rst

```diff
@@ -174,6 +174,28 @@ More details and a description of the package can be found in [5]_.
 
     * :ref:`Receptive Field Estimation and Prediction <sphx_glr_auto_examples_decoding_plot_receptive_field.py>`: Partially replicates the results from Crosse et al. (2016).
 
+Miscellaneous Datasets
+======================
+These datasets are used for specific purposes in the documentation and in
+general are not useful for separate analyses.
+
+ECoG Dataset
+^^^^^^^^^^^^
+:func:`mne.datasets.misc.data_path`. Data exists at ``/ecog/sample_ecog.mat``.
```

Comment:
Fails to render due to not being in `python_reference.rst`, will fix in `master`